src/delegate/index.ios.ts

```diff
@@ -17,35 +12,40 @@ function setup() {
     }
 
     private static getAppDelegate() {
+      // As of NativeScript 8.2, ensureNativeApplication should be called prior to accesses to applicationModule.ios.
+      /* @ts-ignore */
+      if (!!applicationModule.ensureNativeApplication) {
+        /* @ts-ignore */
+        applicationModule.ensureNativeApplication();
+      }
```

Comment:
The explicit coercion to boolean isn't really necessary here, undefined evaluates to falsy anyway; personal preference.

If this library is intended for use for newer TS versions (, the optional chaining operator can be used as well; `applicationModule.ensureNativeApplication?.()`, but while it saves an if check, the syntax is a bit confusing.